systemtests/src/test/java/io/enmasse/systemtest/iot/SimpleK8sDeployTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.enmasse.systemtest.iot;
+
+import static io.enmasse.systemtest.TestTag.sharedIot;
+import static io.enmasse.systemtest.TestTag.smoke;
+
+import java.nio.file.Paths;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.enmasse.systemtest.Environment;
+import io.enmasse.systemtest.Kubernetes;
+import io.enmasse.systemtest.TestUtils;
+import io.enmasse.systemtest.TimeoutBudget;
+import io.enmasse.systemtest.cmdclients.KubeCMDClient;
+
+@Tag(sharedIot)
+@Tag(smoke)
+public class SimpleK8sDeployTest {
+
+    private static final String NAMESPACE = Environment.getInstance().namespace();
+
+    private static final String[] EXPECTED_DEPLOYMENTS = new String[] {
+                    "iot-auth-service",
+                    "iot-device-registry",
+                    "iot-gc",
+                    "iot-http-adapter",
+                    "iot-mqtt-adapter",
+                    "iot-tenant-service",
+    };
+
+    private static final Map<String,String> IOT_LABELS;
+
+    static {
+        IOT_LABELS= new HashMap<> ();
+        IOT_LABELS.put("component", "iot");
+    }
+
+    private Kubernetes client = Kubernetes.getInstance();
+
+    @BeforeAll
+    protected static void setup () throws Exception {
+        KubeCMDClient.createFromFile(NAMESPACE, Paths.get("templates/iot/k8s-systemtests-config.json"));
```

Comment:
Maybe this is the plan, but it should ideally be created using api-model classes instead of putting yaml/json in the systemtests.

Reply:
So right now we still don't have this API model. And it should be easy to replace this later on.

Reply:
I'm working in doing related IoT tests using api-model classes, I'll be creating IoTConfig in api-model soon, so I can also change this when the moment arrives

Reply:
Sounds good to me, thanks @famartinrh